github_ops/git_ops.py

```
@@ -57,7 +62,28 @@ def clone_repo(
     if depth is not None:
         cmd.extend(["--depth", str(depth)])
     logger.info("Cloning %s -> %s", url_or_slug, dest_dir)
-    subprocess.run(cmd, check=True, capture_output=True, text=True)
+    try:
+        subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)